plugin/trino-hive/src/main/java/io/trino/plugin/hive/HivePartitionManager.java

```diff
@@ -154,17 +154,16 @@ public HivePartitionResult getPartitions(ConnectorTableHandle tableHandle, List<
     public HiveTableHandle applyPartitionResult(HiveTableHandle handle, HivePartitionResult partitions, Constraint constraint)
     {
         Optional<List<String>> partitionNames = partitions.getPartitionNames();
-        Optional<List<HivePartition>> partitionList = Optional.empty();
         TupleDomain<ColumnHandle> enforcedConstraint = handle.getEnforcedConstraint();
 
         // Partitions will be loaded if
         // 1. Number of partitionNames is less than or equal to threshold value. Thereby generating additional filter criteria
```

Comment:
`Number of partitionNames` -> `Number of filtered partitionNames` ?
`threshold value` -> let's specify the config name